build/common.sh

```diff
@@ -85,22 +86,26 @@ readonly KUBE_RSYNC_PORT="${KUBE_RSYNC_PORT:-}"
 # mapped to KUBE_RSYNC_PORT via docker networking.
 readonly KUBE_CONTAINER_RSYNC_PORT=8730
 
+# These are the base images for the Docker-wrapped binaries.
+readonly KUBE_GORUNNER_IMAGE="${KUBE_GORUNNER_IMAGE:-$KUBE_BASE_IMAGE_REGISTRY/go-runner:buster-v2.3.1}"
+readonly KUBE_DEBIAN_IPTABLES_IMAGE="${KUBE_DEBIAN_IPTABLES_IMAGE:-$KUBE_BASE_IMAGE_REGISTRY/debian-iptables:buster-v1.5.0}"
+
+# This is the image used in a multi-stage build to apply capabilities to Docker-wrapped binaries.
+readonly KUBE_BUILD_SETCAP_IMAGE="${KUBE_BUILD_SETCAP_IMAGE:-$KUBE_BUILD_SETCAP_REGISTRY/setcap:buster-v1.4.0}"
+
 # Get the set of master binaries that run in Docker (on Linux)
-# Entry format is "<name-of-binary>,<base-image-name>:<base-image-version>".
+# Entry format is "<binary-name>,<base-image>".
 # Binaries are placed in /usr/local/bin inside the image.
-# When building these images the registry for the base images is considered to be ${KUBE_BASE_IMAGE_REGISTRY}.
 #
 # $1 - server architecture
 kube::build::get_docker_wrapped_binaries() {
-  local debian_iptables_version=buster-v1.5.0
-  local go_runner_version=buster-v2.3.1
   ### If you change any of these lists, please also update DOCKERIZED_BINARIES
   ### in build/BUILD. And kube::golang::server_image_targets
   local targets=(
-    "kube-apiserver,go-runner:${go_runner_version}"
-    "kube-controller-manager,go-runner:${go_runner_version}"
-    "kube-scheduler,go-runner:${go_runner_version}"
-    "kube-proxy,debian-iptables:${debian_iptables_version}"
+    "kube-apiserver,${KUBE_GORUNNER_IMAGE}"
```

Comment:
Perhaps it may make more sense to allow overriding the API Server image etc.? E.G. In the future we may decide to move off of go runner or debian-iptables.

Reply:
+1 to `KUBE_APISERVER_BASE_IMAGE`, etc.

Would also recommend explicitly commenting by the per-binary image definitions that this allows `make` callers to override base images.

Reply:
sgtm

Reply:
> Would also recommend explicitly commenting by the per-binary image definitions that this allows make callers to override base images.

done